sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/OperatorStateMetadataSuite.scala

```diff
@@ -32,30 +32,37 @@ class OperatorStateMetadataSuite extends StreamTest with SharedSparkSession {
 
   private lazy val hadoopConf = spark.sessionState.newHadoopConf()
 
+  private var checkpointDir = Utils.createTempDir()
+
   private def numShufflePartitions = spark.sessionState.conf.numShufflePartitions
 
-  private def sameOperatorStateMetadata(
-      operatorMetadata1: OperatorStateMetadataV1,
-      operatorMetadata2: OperatorStateMetadataV1): Boolean = {
-    operatorMetadata1.operatorInfo == operatorMetadata2.operatorInfo &&
-      operatorMetadata1.stateStoreInfo.sameElements(operatorMetadata2.stateStoreInfo)
+  override def beforeEach(): Unit = {
+    Utils.deleteRecursively(checkpointDir)
+    checkpointDir = Utils.createTempDir()
+  }
+
+  private def checkOperatorStateMetadata(
+      operatorId: Int,
+      expectedMetadata: OperatorStateMetadataV1): Unit = {
+    val statePath = new Path(checkpointDir.toString, s"state/$operatorId")
+    val operatorMetadata = new OperatorStateMetadataReader(statePath, hadoopConf).read()
+      .asInstanceOf[OperatorStateMetadataV1]
+    // println("doodoo: " + operatorMetadata.stateStoreInfo)
```

Comment:
nit: should be removed

Reply:
Sorry, forgot to delete the debug print.